sherpa/csrc/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(sherpa_srcs
 )
 
 add_library(sherpa_core ${sherpa_srcs})
-target_link_libraries(sherpa_core PUBLIC ${TORCH_LIBRARIES})
+target_link_libraries(sherpa_core PUBLIC ${TORCH_LIBRARIES} ${K2_LIBRARIES})
```

Comment:
This line shows how to link pre-compiled k2 libraries.
It also sets up the include path for `k2/torch_api.h`.